### DISCUSSION TITLE
Umfrage: Welche Version von GEN24_Ladesteuerung benutzt du, Modbus oder HTTP?

### DISCUSSION THREAD

**Discussion by wiggal** [wiggal/GEN24_Ladesteuerung#83] [Polls]
Da ich mir nicht sicher bin, inwieweit ich die Modbus Version weiter pflegen werde, hier eine kleine Umfrage, wer welche Version verwendet. 
Gerne könnt ihr hier auch eure Meinung dazu hinterlassen.

**Comment by mrfreezer** [comment]
mir ist nicht ganz klar, welche nachteile die http-version hat, habe jetzt aber auf sie umgeschwenkt zugunsten der dann nicht außer kraft gesetzten automatismen im wechselrichter 
(namentlich produktionsüberschuss > einspeisebegrenzung / ac-leistung -> akku)

**Comment by wiggal** [comment]
Ok, nachdem in die HTTP-Version alle Funktionen übernommen wurden, werde ich die Modbus-Version am 0.25 **NICHT** mehr weiter entwickeln!!!
Laut Umfrage wird sie ja sowieso nicht mehr verwendet.

Wiggal

**Comment by wiggal** [comment]
Da die 
mobus Version nicht mehr gepflegt wird, Umfage erledigt
